openwisp_users/base/models.py

```diff
@@ -78,13 +78,19 @@ def organizations_pk(self):
         return qs
 
     def is_member(self, organization):
+        if self.is_superuser:
+            return True
         return str(organization.pk) in self.organizations_dict
 
     def is_manager(self, organization):
+        if self.is_superuser:
+            return True
         org_dict = self.organizations_dict.get(str(organization.pk))
         return org_dict is not None and (org_dict['is_admin'] or org_dict['is_owner'])
 
     def is_owner(self, organization):
+        if self.is_superuser:
+            return True
```

Comment:
superuser can be owner, manager, member at will, so always return `True`.

Reply:
superuser can do anything and these checks do not apply to them, `if user.is_superuser` should always be checked beforehand using these methods